ubuntu.22.04/Dockerfile

```diff
@@ -198,3 +194,6 @@ RUN curl -sSL -o argocd-linux-amd64 https://github.com/argoproj/argo-cd/releases
 RUN add-apt-repository -y ppa:rmescandon/yq && \
     apt-get update && apt-get install -y jq yq openssh-client rsync git augeas-tools && \
     rm -rf /var/lib/apt/lists/*
+
+# remove az cli warning - https://github.com/Azure/arm-deploy/issues/173
+RUN az config set bicep.use_binary_from_path=false
```

Comment:
I'm not sure we should have a separate container layer just for this. Is it worth putting it in the batched command for the Az CLI install instead?

Reply:
Line 92/93 for reference

Reply:
The layers was the reason that I put it down here on its own.
As far as I understand, changing an earlier command/layer invalidates everything that comes after it, this would allow all other previously cached layers to be re-used, but I could be wrong.